libs/game/sprite.ts

```diff
@@ -69,7 +69,7 @@ class Sprite extends sprites.BaseSprite {
     //% group="Physics" blockSetVariable="mySprite"
     //% blockCombine block="x" callInDebugger
     get x(): number {
-        return Fx.toInt(this._x) + (this._image.width >> 1)
+        return Fx.toFloat(this._x) + (this._image.width >> 1)
```

Comment:
```suggestion
        return Fx.toFloat(this._x) + (this._image.width / 2)
```
So that we're floats all the way?